pyrene/src/components/TreeTable/TreeTable.tsx

```diff
@@ -26,7 +28,8 @@ export interface TreeTableProps<R>{
    * Sets the Table data displayed in the rows.
    * Type: [{ children: object, lineCount: number, ...row }]
    */
-  data: Array<RowData<R>>,
+  data: Array<R>,
+  defaultExpandedSection?: string,
```

Comment:
I think this is a public component, so can we also have a description/comment for this prop?

Reply:
> I think this is a public component, so can we also have a description/comment for this prop?

The props `defaultExpandedSection` is actually not used in the `TreeTable` component. So I just removed it. Thanks @cyrilwanner